theano/gpuarray/opt.py

```diff
@@ -135,9 +180,10 @@ def local_opt(node):
                         context_name = i.owner.inputs[0].type.context_name
                         replace = True
                         break
+                clients = [c for o in node.outputs for c in o.clients]
```

Comment:
Yes, earlier, if you remember, we passed clients also to few optimizers before outputs were also passed along. Now it can be moved back